testdata/workspaceConfigTemplate.json

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
Are the keys / ids in this file dummy?

Reply:
No,  Key and id are from app.dev.rudderlabs.com/env